### ISSUE TITLE
links to code snippets

### ISSUE THREAD

**Issue by rabernat** [ocean-eddy-cpt/gcm-filters#14]
I have a few useful code snippets floating around that I want to link here.

- My notebook with a simple 5-point stencil that shows the full workflow: https://nbviewer.jupyter.org/rabernat/429238a9db40fb3dbfc74b84f64a4b36
- Example of evaluating Laplacian / biharmonic from CESM POP: https://nbviewer.jupyter.org/gist/rabernat/d1806ae5e38a292e62a45ee172f661c4
   Note that this one _does not_ use the GPU but does use Xarray / Dask. It can be run from Pangeo cloud and uses the [cloud-based POP data](https://catalog.pangeo.io/browse/master/ocean/CESM_POP/).
- A very rough notebook which sketches how you would do the POP example on the GPU: https://nbviewer.jupyter.org/gist/rabernat/a6f2a836ee9acecd281fea8c91196a6f
   This uses dummy data, rather than real POP data. I ran it on Casper.

cc @LaureZanna

**Comment by jbusecke** [comment]
The first link does not work for me. 

Should we make those notebooks part of the docs? #11 is ready for merge and I could migrate those over.

**Comment by rabernat** [comment]
No, they are just notes for reference to help with implementation. They don't belong in the docs.

Our docs should describe how to use gcm-filters, NOT how it is implemented.
